### PR TITLE
#6744 follow-up: grammar & whitespace adjustments

### PR DESCRIPTION
**Pull request by illfated**
**Description:**

The PR #6744 added a note text line to the page, but without the common Note blob used in the default MS Docs code style.

Commit https://github.com/MicrosoftDocs/windows-itpro-docs/commit/083e35c01c2020403ec97aa952b0880ac32fb2c1 by @garycentric corrected this line to a Note blob.

This PR (after the very recent changes) now only corrects some minor note text grammar and makes a few whitespace adjustments.

**Changes proposed:**
- Remove redundant end-of-line whitespace (blanks) throughout the page
- Add 1 blank line between the metadata section and the page title
- Grammar corrections (https://github.com/MicrosoftDocs/windows-itpro-docs/pull/6768#discussion_r428151193) by the document author

**Ticket closure or reference:**

Ref. PR #6744 and commit https://github.com/MicrosoftDocs/windows-itpro-docs/commit/083e35c01c2020403ec97aa952b0880ac32fb2c1

### CODE REVIEW

**Comment by illfated** [comment]
#sign-off

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-dir-sync.md [hunk 1]
```diff
@@ -61,14 +62,15 @@ Sign-in a domain controller or management workstation with _Domain Admin_ equiva
 5. In the **Enter the object names to select** text box, type the name of the Azure AD Connect service account.  Click **OK**.
 6. Click **OK** to return to **Active Directory Users and Computers**.
 
-Note: if your AD forest has multiple domains. Please make sure you add the ADConnect sync service account (ie. MSOL_12121212) into "Enterprise Key Admins" group to gain permission across the domains in the forest.
+> [!NOTE]
+> if your AD forest has multiple domains. Please make sure you add the ADConnect sync service account (ie. MSOL_12121212) into "Enterprise Key Admins" group to gain permission across the domains in the forest.
```
Comment (mapalko) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-dir-sync.md:66]: Grammar in this note also needs to be fixed.
```suggestion
> If your AD forest has multiple domains, make sure you add the ADConnect sync service account (ie. MSOL_12121212) into "Enterprise Key Admins" group to gain permission across the domains in the forest.
```

Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-dir-sync.md:66]: You are correct. Thank you for noticing. Also thanks for making a commit suggestion, so I can merge the commit before leaving the office, without having to go via Git Extensions.

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
Please note that this PR has now been updated and changed, both in the title and in the main PR comment, to reflect the changes yesterday in commit https://github.com/MicrosoftDocs/windows-itpro-docs/commit/083e35c01c2020403ec97aa952b0880ac32fb2c1 (by @garycentric) and the added grammar correction provided by @mapalko in https://github.com/MicrosoftDocs/windows-itpro-docs/pull/6768#discussion_r428151193.

**Comment by e0i** [comment]
@Dansimp The PR can be merged as has been approved by the article author, Matthew P. Thank you.

**Pull request merged**